packages/ui-kit-components/src/components/IndexTickers/TickerBar.tsx

```diff
@@ -0,0 +1,44 @@
+import { Box, useMediaQuery, useTheme } from '@mui/material';
+
+import IndexTickers from './IndexTickers';
+
+const TickerBar: React.FC = () => {
+  const matches = useMediaQuery('(min-width: 830px)');
+  const { palette } = useTheme();
+  return (
+    <Box
+      pt={2}
+      borderTop={1}
+      width={'100%'}
+      borderColor={palette.divider}
+      display={'flex'}
+      justifyContent={'center'}
+    >
+      <IndexTickers
+        includeDivider={false}
+        name={'SPA'}
```

Comment:
Wouldn't it be better to accept props for the name, latest price, change price, and percentage?  That way we could customize the stocks we want to display, and we could use live data instead.